rmw_fastrtps_shared_cpp/src/rmw_node.cpp

```diff
@@ -327,6 +346,27 @@ __rmw_create_node(
         Property(
           "dds.sec.access.builtin.Access-Permissions.permissions", security_files_paths[5]));
 
+      std::string security_logging_file_path;
+      if (get_security_file_path(security_logging_file_path,
+                                 "logging.xml", security_options->security_root_path)) {
```

Comment:
You are right, I didn't run the ROS 2 linters here. I'll see that done asap.

At the moment the security logging is meant to be configured at the *participant* level (currently node, soon context). We do not plan on widening the application of a given config here but this is something we envision directly at the launchfile level - e.g. a single flag `--sec-logging` enables a default config for a whole launch file / or a bunch of options in the launchfile etc. It will likely be part of the feature set that NoDL will bring (altho it is not directly related to it).